app/javascript/dashboard/components/ui/TimeAgo.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <span class="time-ago">
+    <span> {{ timeAgo }}</span>
+  </span>
+</template>
+
+<script>
+const ZERO = 0;
+const MINUTE_IN_MILLI_SECOND = 60000;
+const HOUR_IN_MILLI_SECOND = MINUTE_IN_MILLI_SECOND * 60;
+const DAY_IN_MILLI_SECOND = HOUR_IN_MILLI_SECOND * 24;
+const MONTH_IN_MILLI_SECOND = DAY_IN_MILLI_SECOND * 30;
+const YEAR_IN_MILLI_SECOND = MONTH_IN_MILLI_SECOND * 12;
+
+import timeMixin from 'dashboard/mixins/time';
+import { differenceInMilliseconds } from 'date-fns';
+
+export default {
+  name: 'TimeAgo',
+  mixins: [timeMixin],
+  props: {
+    isAutoRefreshEnabled: {
+      type: Boolean,
+      default: true,
+    },
+    timestamp: {
+      type: [String, Date, Number],
+      default: '',
+    },
+  },
+  data() {
+    return {
+      timeAgo: '',
+      timer: null,
+    };
+  },
+  computed: {
+    refreshTime() {
+      const timeDiff = differenceInMilliseconds(
+        new Date(),
+        new Date(this.timestamp * 1000)
+      );
+      if (
+        timeDiff > MINUTE_IN_MILLI_SECOND &&
+        timeDiff < HOUR_IN_MILLI_SECOND
+      ) {
+        return MINUTE_IN_MILLI_SECOND;
+      }
+      if (timeDiff > HOUR_IN_MILLI_SECOND && timeDiff < DAY_IN_MILLI_SECOND) {
+        return HOUR_IN_MILLI_SECOND;
+      }
+      if (timeDiff > DAY_IN_MILLI_SECOND && timeDiff < MONTH_IN_MILLI_SECOND) {
+        return DAY_IN_MILLI_SECOND;
+      }
+      if (timeDiff > MONTH_IN_MILLI_SECOND && timeDiff < YEAR_IN_MILLI_SECOND) {
+        return MONTH_IN_MILLI_SECOND;
+      }
+      if (timeDiff > YEAR_IN_MILLI_SECOND) {
+        return YEAR_IN_MILLI_SECOND;
+      }
+      return ZERO;
```

Comment:
```suggestion
      if (timeDiff > YEAR_IN_MSEC) {
        return ZERO;
      }
      if (timeDiff > MONTH_IN_MSEC) {
        return ZERO;
      }
      if (timeDiff > DAY_IN_MSEC) {
        return DAY_IN_MSEC;
      }
      if (timeDiff > HOUR_IN_MSEC) {
        return HOUR_IN_MSEC;
      }
      if (timeDiff > MINUTE_IN_MSEC) {
        return MINUTE_IN_MSEC;
      }

      return ZERO;
```
@iamsivin Please test this suggested change working or not?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/5253/commits/d9cf0b645453499a7d6cc0ea4cfee402f48f5420